doc/using_tf.rst

```diff
@@ -133,6 +133,34 @@ In your training script the channels will be stored in environment variables ``S
    ``output_path``.
 
 
+Use third-party libraries
+-------------------------
+
+If there are other packages you want to use with your script, you can use a ``requirements.txt`` to install other dependencies at runtime.
+
+For training, support for installing packages using ``requirements.txt`` varies by Tensorflow version as follows:
+
+- For TensorFlow 1.11 or newer using Script Mode without Horovod, Tensorflow 1.15.2 with Python 3.7 or newer, and Tensorflow 2.2 or newer:
+    - Include a ``requirements.txt`` file in the same directory as your training script.
+    - You must specify this directory using the ``source_dir`` argument when creating a Tensorflow estimator.
+- For older versions of Tensorflow using Script Mode with Horovod:
+    - Write a shell script for your entry point that first calls ``pip install -r requirements.txt``, then runs your training script.
+    - For an example of using shell scripts, see `this example notebook <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/tensorflow_script_mode_using_shell_commands/tensorflow_script_mode_using_shell_commands.ipynb>`__.
+- For older versions of Tensorflow using Legacy Mode:
```

Comment:
QQ :
If I am using framework_version = '1.15.0'  and py36 , what is the way to install additional packages?

Reply:
all TensorFlow Python 3 images use Script Mode, so it would depend on whether or not you're using Horovod. If so, then this falls under "For older versions of Tensorflow using Script Mode with Horovod". Otherwise, this falls under "For TensorFlow 1.11 or newer using Script Mode without Horovod"